### PR TITLE
Mobile layout

### PR DESCRIPTION
**Pull request by DuongTyler**

### CODE REVIEW

**Comment by DuongTyler** [comment]
[Fix mobile view](https://trello.com/c/I0hGk0AH/96-fix-mobile-view)

File: frontend/src/App.css [hunk 1]
```diff
@@ -2,6 +2,28 @@ body {
     background-color: #282c34;
 }
 
+.notice-container {
+		background-image: url(assets/cards/card2.png);
+		background-size: cover;
+    border-radius: 15px;
```
Comment (zapplecat) [inline frontend/src/App.css:8]: nit: indentation

File: frontend/src/App.css [hunk 2]
```diff
@@ -2,6 +2,28 @@ body {
...
+		text-align: center;
+		margin: 2rem;
+		max-width: 800px;
+}
+
+.notice-center {
+    display: flex;
```
Comment (zapplecat) [inline frontend/src/App.css:15]: nit: indentation

File: frontend/src/App.css [hunk 3]
```diff
@@ -16,8 +38,16 @@ body {
...
+@media (max-height: 800px) {
+		/*Handle mobile layouts since everything is freaking tiny*/
+		.community-message-card {
+				margin-top: 12vh;
+		}
+}
+
 @media (max-width: 900px) {
     .community-message-card {
+				margin-top: 25vh;
```
Comment (zapplecat) [inline frontend/src/App.css:50]: nit: tabs were used instead of spaces

File: frontend/src/App.css [hunk 4]
```diff
@@ -55,14 +85,21 @@ body {
...
   right: 0;
   margin: auto;
 
   transition: "all 0.25s ease";
 }
 
+@media (max-height: 900px) {
+		/*If we can't get the damn thing to stay off card text, just hide the thing*/
+		.anchor-link {
+				opacity: 0;
```
Comment (zapplecat) [inline frontend/src/App.css:99]: nit: tabs used instead of spaces

File: frontend/src/components/navbar.css [hunk 5]
```diff
@@ -1,6 +1,6 @@
 .navbar {
     width: 100%;
-    height: 75px;
+    /*height: 75px;*/
```
Comment (zapplecat) [inline frontend/src/components/navbar.css:3]: Delete if we don't need this?

File: frontend/src/components/navbar.css [hunk 6]
```diff
@@ -11,6 +11,7 @@
     padding: 0 5vw;
 
     display: flex;
+		flex-wrap: wrap;
```
Comment (zapplecat) [inline frontend/src/components/navbar.css:14]: nit: used tab instead of spaces

**Pull request merged**